src/screens/QuickUpdateScreen/HeaderFilterButton/HeaderFilterButton.js

```diff
@@ -0,0 +1,59 @@
+import React, { PureComponent } from 'react';
```

Comment:
this is been taken and turned into `SelectMenu` -- after merging/rebasing master -- you should probably use that and make sure it supports functionality you expect.